src/main/java/com/minecraftabnormals/savageandravage/common/entity/block/SporeBombEntity.java

```diff
@@ -14,6 +10,9 @@
 import net.minecraft.world.World;
 import net.minecraftforge.fml.network.NetworkHooks;
 
+import javax.annotation.Nonnull;
```

Comment:
Unused imports

Reply:
I can optimize imports after all requested changes have been applied to make it easier if you like.

Reply:
Works for me